app/views/albums/new.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Albums#new</h1>
+<p>Find me in app/views/albums/new.html.erb</p>
```

Comment:
We should take these lines out, as they're the default text given by Rails to indicate to us (not the user) where we can find the file so we can put our own content into it instead.